app.rb

```diff
@@ -118,11 +119,11 @@
 # because wget simply fetches the HTML document, this lets us continue to
 # spider the site to generate the contents of everypolitician/viewer-static.
 # See scripts/release.sh (update_viewer_static).
-def docs_redirect(path, page_title)
-  url = URI.join(settings.docs_url, path)
+def redirect(host, path, page_title)
```

Comment:
Calling this `redirect` scares me a little, as it's doing something different from what most people are likely to think a `redirect` method would do. I think it would be better to make it more explicitly `soft_redirect` or `meta_redirect` or somesuch.